radix-engine/src/transaction/transaction_executor.rs

```diff
@@ -94,6 +95,11 @@ impl ExecutionConfig {
         self
     }
 
+    pub fn with_cost_unit_limit(mut self, cost_unit_limit: u32) -> Self {
+        self.cost_unit_limit = cost_unit_limit;
+        self
+    }
```

Comment:
But yes, I guess there might be separate parameters:
* Static cost unit limit which we expose to Scrypto
* "Abort after" cost unit limit which is just used to limit the engine.

This parameter is just supposed to be that static cost unit limit - I only allow this to be changed because it's overridden in some tests :)